src-electron/util/types.js

```diff
@@ -276,20 +278,13 @@ function isFloat(type) {
  * @param {*} type
  * @returns true if type is signed integer, false otherwise
  */
-function isSignedInteger(type) {
-  switch (type) {
-    case 'int8s':
-    case 'int16s':
-    case 'int24s':
-    case 'int32s':
-    case 'int40s':
-    case 'int48s':
-    case 'int56s':
-    case 'int64s':
-      return true
-    default:
-      return false
-  }
+async function isSignedInteger(db, sessionId, type) {
+  let sessionPackages = await queryPackages.getSessionPackages(db, sessionId)
+  return await queryAtomic.isAtomicSignedByNameAndPackage(
```

Comment:
You do not need this new function we have code to retrieve this using query-zcl.js#selectNumberByName

Reply:
oh thanks